exporters/sender/jdk/src/test/java/io/opentelemetry/exporter/sender/jdk/internal/JdkHttpSenderTest.java

```diff
@@ -66,6 +78,28 @@ void setup() throws IOException, InterruptedException {
                 .build());
   }
 
+  @SuppressWarnings("SystemOut")
+  @Test
+  void exportAsJson() {
+    HttpExporter<Marshaler> exporter =
+        new HttpExporterBuilder<>("jdk", "test", server.httpUri().toASCIIString())
+            .setAuthenticator(
+                () -> {
+                  Map<String, String> headers = new HashMap<>();
+                  headers.put("Authorization", "auth");
+                  return headers;
+                })
+            .exportAsJson()
+            .build();
+
+    server.enqueue(HttpResponse.of(HttpStatus.OK));
+
+    CompletableResultCode result = exporter.export(new NoOpMarshaler(), 0);
+    result.join(1, TimeUnit.MINUTES);
+    exporter.shutdown().join(1, TimeUnit.MINUTES);
+    Assertions.assertThat(result.isSuccess()).isTrue();
```

Comment:
Not sure what this test has to do with the code we've been talking about.

Reply:
I intend to cover Lines 220 and 221 of JdkHttpSender after a successful response. I can't override the method writeJsonTo on the NoOpMarshaler so I'm using a succes flow instead.

Reply:
Just removed the authentication and shutdown part to make it slightly smaller